template/blast.config.js

```diff
@@ -3,18 +3,8 @@ module.exports.config = {
   name: 'template',
   network: 'cudos',
   defaultAccount: 'account1',
-  accounts: {
-    'account1':
-    {
-      address: 'cudos1yvtuaadhfhxf8ke7zm902z4rj382a8ayymq32s',
-      mnemonic: 'ordinary witness such toddler tag mouse helmet perfect venue eyebrow upgrade rabbit'
-    },
-    'account2':
-    {
-      address: 'cudos1xgejtl9heykzscska6cetsferpelygx8g9vxp9',
-      mnemonic: 'course hurdle stand heart rescue trap upset cousin dish embody business equip'
-    }
-  },
+  additionalCustomAccounts: 0, // default is 10
```

Comment:
In my opinion this variable can be named "additionalAccounts" or "customAccounts" vs "defaultAccount".

Also this comment seems a little confusing - "default accounts are 10" seems better or you can also move it on line 5